addons/purchase/views/purchase_views.xml

```diff
@@ -322,6 +322,7 @@
                                      <field name="price_tax"/>
                                      <field name="price_total"/>
                                      <field name="price_unit"/>
+                                     <field name="discount" string="Discount:"/>
```

Comment:
Do you need the string here?

Reply:
I added it so it would be exported to the translation `.pot`, not sure if there's another way to do that.

Reply:
It will be exported from where you added it within the `<span>` :wink:

Reply:
Yep, seems to work. Guess I hadn't added the `<span>` tags yet when I first tried it